### ISSUE TITLE
Base R version

### ISSUE THREAD

**Issue by tfalcs** [ohdsi-studies/HERACharacterization#2]
Hey @anthonysena ,

I just opened the R project for HERA and it immediately gives a warning about the project requesting R 3.6.3 but I am running 4.0.2. I had to upgrade my base R for the DLMM OHDSI study Jenna and Martijn are managing (package dependencies required it). Can I run this study on 4.0.2?

**Comment by anthonysena** [comment]
Yes - the package will work with R v3.6 or v4. Stick with 4 if you’re already upgraded.

Sent from my iPhone

On Oct 13, 2020, at 9:47 AM, tfalcs <notifications@github.com> wrote:

﻿
WARNING: This email originated from outside the company. Do not click on links unless you recognize the sender and have confidence the content is safe. If you have concerns about this email, send it as an attachment to ‘SuspiciousEmail@ITS.JNJ.COM’.


Hey @anthonysena<https://github.com/anthonysena> ,

I just opened the R project for HERA and it immediately gives a warning about the project requesting R 3.6.3 but I am running 4.0.2. I had to upgrade my base R for the DLMM OHDSI study Jenna and Martijn are managing (package dependencies required it). Can I run this study on 4.0.2?
